src/openrpc/lifecycle.json

```diff
@@ -1,379 +1,626 @@
+            "params": [
+                {
+                    "name": "reason",
+                    "summary": "The reason the app is requesting to be closed",
+                    "required": true,
+                    "schema": {
+                        "$ref": "https://meta.rdkcentral.com/firebolt/schemas/lifecycle#/definitions/CloseReason"
+                    }
+                }
+            ],
+            "result": {
+                "name": "success",
+                "schema": {
+                    "const": null
+                }
+            },
+            "examples": [
+                {
+                    "name": "Close the app when the user presses back on the app home screen",
+                    "params": [
+                        {
+                            "name": "reason",
+                            "value": "remoteButton"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                },
+                {
+                    "name": "Close the app when the user selects an exit menu item",
+                    "params": [
+                        {
+                            "name": "reason",
+                            "value": "userExit"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
         },
         {
-          "name": "capabilities",
-          "x-uses": ["xrn:firebolt:capability:lifecycle:state"]
-        }
-      ],
-      "summary": "Notify the platform that the app is done unloading",
-      "params": [],
-      "result": {
-        "name": "results",
-        "schema": {
-          "const": null
-        }
-      },
-      "examples": [
-        {
-          "name": "Default Example",
-          "params": [],
-          "result": {
-            "name": "Default Result",
-            "value": null
-          }
-        }
-      ]
-    },
-    {
-      "name": "state",
-      "summary": "Get the current state of the app. This function is **synchronous**.",
-      "tags": [
-        {
-          "name": "synchronous"
+            "name": "start",
+            "tags": [
+                {
+                    "name": "notifier",
+                    "x-notifier": "Lifecycle.start",
+                    "x-event": "Lifecycle.onStart"
+                },
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:lifecycle:state"
+                    ]
+                }
+            ],
+            "summary": "Listen to the start event",
+            "params": [
+                {
+                    "name": "listen",
```

Comment:
notifiers don't have a `listen` param, they just have the event payload(s). These events all have no payloads except for `destroy`